CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ set(VERSION_MAJOR 1)
 set(VERSION_MINOR 0)
 set(VERSION_PATCH 0)
 
+option(DLIBXX_ENABLE_TESTS OFF "Enable testing")
```

Comment:
Can you add a comment for how to enable and run the tests? Also add a section at the bottom of the README with similar instructions.

Reply:
Will do